urlhaus/info.json

```diff
@@ -0,0 +1,385 @@
+{
+    "name": "urlhaus",
+    "version": "1.0.1",
+    "label": "Url Haus",
+    "description": "URLhaus is a project operated by abuse.ch. The purpose of the project is to collect, track and share malware URLs, helping network administrators and security analysts to protect their network and customers from cyber threats.",
+    "publisher": "Community",
+    "icon_small_name": "urlhaus.png",
+    "icon_large_name": "urlhaus.png",
+    "category": "Threat Intelligence",
+    "configuration": {
+        "fields": [
+            {
+                "title": "Server URL",
+                "type": "text",
+                "name": "server_url",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "value": "https://urlhaus-api.abuse.ch"
+            }
```

Comment:
Verify SSL parameter missing in configuration parameter.

Reply:
no need due to validate internet connectivity only